granary/microformats2.py

```diff
@@ -293,6 +295,17 @@ def json_to_object(mf2):
     'tags': [json_to_object(cat) for cat in props.get('category', [])],
   }
 
+  lat, lng = prop.get('latitude'), prop.get('longitude')
+  if lat and lng:
+    try:
+      lat, lng = float(lat), float(lng)
+      # 2 leading places for latitude, 3 for longitude
+      obj['position'] = '%0+10.6f%0+11.6f/' % (lat, lng)
```

Comment:
make `postprocess_object` (and/or `_activity`) a class method and call them like that? looks like we currently override them in `googleplus`, but those could be class methods too. (the `self.actor_name()` call in `postprocess_activity` is misleading, it's actually a static method.)

alternatively you're welcome to just add the TODO and keep the `position` code here. up to you!